web-client/public/js/updateapp.js

```diff
@@ -0,0 +1,5 @@
+/* eslint no-unused-vars: [2, {"varsIgnorePattern": "setupHandlers|grnState"}] */
+var setupHandlers = require("./setupHandlers");
+export var updateApp = function (grnState) {
+    return grnState;
```

Comment:
`updateApp` is supposed to set `$("#dashedGrayLineButton").prop("checked")` according to what `grnState.dashedLine` is.